packages/utils/src/metrics/index.ts

```diff
@@ -0,0 +1,12 @@
+import {Gauge, Histogram} from "prom-client";
```

Comment:
Do not import prom-client, define a look-a-like type like here 
https://github.com/ChainSafe/lodestar/blob/unstable/packages/reqresp/src/metrics.ts